variables.tf

```diff
@@ -86,4 +80,10 @@ variable "tags" {
   description = "A mapping of tags to assign to all resources"
   type        = map(string)
   default     = {}
-}
\ No newline at end of file
+}
+
+variable "dlq_options" {
+  description = "Boolean to create DLQ queue"
+  type        = map(string)
```

Comment:
This should be map(any) since we have integers.

Reply:
Make it an object and specify default values here like we do with s3 using experimental options

Reply:
https://github.com/variant-inc/terragrunt-variant-apps/blob/0fd68d30da8cce7b4e4963fa5ac7f270541b49ed/modules/common/buckets/variables.tf#L26 For specifying which ones are optional

Reply:
```
local.dlq_options = defaults(var.dlqOptions, { 
visibility_timeout = 30,
message_retention_seconds = 1209600

 })```

Reply:
Setting it to null will set the default values form the module https://github.com/terraform-aws-modules/terraform-aws-sqs/blob/master/variables.tf#L19

Reply:
Added as requested.

Reply:
As discussed, changed back to lookup as it was causing other problems.